### ISSUE TITLE
Ideas from SixDegrees

### ISSUE THREAD

**Issue by hartenthaler** [Neriderc/GVExport#151]
Do you know the wikitree app SixDegrees? Maybe there are some ideas that can be used for GVExport, too.
https://apps.wikitree.com/apps/clarke11007/SixDegrees.php
https://youtu.be/1Ragqcg6QHM

- option to remove all information in boxes beside the person's name
- show short versions of the names (initials, first name only)
- don't use the update button (all changes are visible instantaneously)
- allow the user to select some fonts
- ...

**Comment by Neriderc** [comment]
I did not know about SixDegrees! It looks very nice, and those are some good ideas.

>    option to remove all information in boxes beside the person's name

I didn't realise until I tested just now, but the option to show birth date/death date does not work as I expected. I expected if you unchecked all these options, only the name would be left. However, when I test this, it leaves the */+ making it looks like an unknown date. I think we should update this setting to completely remove this. Unselecting all these options should lead to just a name and nothing else shown.

> show short versions of the names (initials, first name only)

And perhaps, like in SixDegrees, showing no name at all? We could have name options - Initials, First initial and last name, First name only, Last name only, Don't show names.

>     don't use the update button (all changes are visible instantaneously)

I'm trying to think of down sides to this. One is that it can take many seconds to complete a large tree. Triggering this every time you change a setting could be a problem on older computers. Perhaps we could implement this but have an advanced option to disable it?

We have discussed advanced options before, I'm thinking a new section at the bottom that is hidden until you click to open it, with these kinds of setting in it.

>   allow the user to select some fonts

This could be quite simple to implement so long as we stick to "web safe" fonts. There is already a (hidden) setting for font, we could add this in an "advanced" section as above.

Thanks for the ideas!
